internal/query/org.go

```diff
@@ -12,17 +12,60 @@ import (
 	"github.com/caos/zitadel/internal/query/projection"
 )
 
+var (
+	orgsTable = table{
+		name: "zitadel.projections.orgs",
+	}
+	OrgColumnID = Column{
+		name:  projection.OrgColumnID,
+		table: orgsTable,
+	}
+	OrgColumnCreationDate = Column{
+		name:  projection.OrgColumnCreationDate,
+		table: orgsTable,
+	}
+	OrgColumnChangeDate = Column{
+		name:  projection.OrgColumnChangeDate,
+		table: orgsTable,
+	}
+	OrgColumnResourceOwner = Column{
+		name:  projection.OrgColumnResourceOwner,
+		table: orgsTable,
+	}
+	OrgColumnState = Column{
+		name:  projection.OrgColumnState,
+		table: orgsTable,
+	}
+	OrgColumnSequence = Column{
+		name:  projection.OrgColumnSequence,
+		table: orgsTable,
+	}
+	OrgColumnName = Column{
+		name:  projection.OrgColumnName,
+		table: orgsTable,
+	}
+	OrgColumnDomain = Column{
+		name:  projection.OrgColumnDomain,
+		table: orgsTable,
+	}
+	OrgsColumnCount = Column{
+		name:  "COUNT(*) OVER ()",
+		table: orgsTable,
+	}
+)
+
 func prepareOrgQuery() (sq.SelectBuilder, func(*sql.Row) (*Org, error)) {
 	return sq.Select(
-			OrgColumnID.toFullColumnName(),
-			OrgColumnCreationDate.toFullColumnName(),
-			OrgColumnChangeDate.toFullColumnName(),
-			OrgColumnResourceOwner.toFullColumnName(),
-			OrgColumnState.toFullColumnName(),
-			OrgColumnSequence.toFullColumnName(),
-			OrgColumnName.toFullColumnName(),
-			OrgColumnDomain.toFullColumnName()).
-			From(projection.OrgProjectionTable).PlaceholderFormat(sq.Dollar),
+			OrgColumnID.sql(),
```

Comment:
I would preferre to name this function different than sql(), for me its not clear what it does

Reply:
renamed to identifier